gpslogger/src/main/res/values/styles.xml

```diff
@@ -47,6 +47,8 @@
         <item name="android:windowCloseOnTouchOutside">true</item>
     </style>
 
+    <!--    Automatically gets applied to preference screens, equivalent to: -->
+    <!--    <item name="preferenceTheme">@style/PreferenceThemeOverlay</item>-->
     <style name="PreferenceThemeOverlay">
```

Comment:
What I meant is that all of this can be removed as well :)

Reply:
Oh I see.  When I remove all of it, the styles on the preference screens get weird.  

Here's without this block

![image](https://user-images.githubusercontent.com/746276/135576109-37472888-f157-486e-b014-5e613b60d166.png)

And with this block

![image](https://user-images.githubusercontent.com/746276/135576158-8032921a-2acd-4943-a9ee-af2bceed0ee1.png)

There are [workarounds](https://stackoverflow.com/questions/18509369/android-how-to-remove-margin-padding-in-preference-screen) to removing the strange padding, but they were tedious or hacky. I think I had simply copied [this snippet](https://stackoverflow.com/questions/34220035/how-to-style-preferencefragmentcompat) and liked the 'larger' style.

Reply:
I also tried just this

```
    <style name="PreferenceThemeOverlay">

        <item name="iconSpaceReserved">false</item>
    </style>
```

Then it gets weirder as some headings become large and some are small.  

![image](https://user-images.githubusercontent.com/746276/135577972-d0a96f86-c12e-4890-80a0-ed32aeb1a206.png)

So it was easiest to just leave this block in place and not think about it.

![image](https://user-images.githubusercontent.com/746276/135578048-7921a34f-e408-4227-9310-b43fdcc644ae.png)

Reply:
Oh, I can fix this.